### ISSUE TITLE
鼠须管找不到地方禁用通知

### ISSUE THREAD

**Issue by GoogleCodeExporter** [lotem/rimeime#328]
```
請寫下能夠重現此狀況的操作步驟：
1. 切换中英文

所得的結果是這個樣：
右上角出现状态通知

預期結果該是那個樣：
希望能在配置中禁用这个通知，非常非常非常annoying

請提供以下信息，幫我們定位問題。

1. 輸入法的款式：鼠鬚管 
   輸入法版本號：
   distribution_version: 0.9.12
   install_time: "Thu Dec 13 10:24:30 2012"
   rime_version: 0.9.6
   update_time: "Tue Dec 25 00:09:27 2012"

2. 操作系統類型：Mac
   操作系統版本：ML

3. 輸入方案名稱：朙月拼音

還想再說幾句：


```

Original issue reported on code.google.com by `5h3l...@gmail.com` on 25 Dec 2012 at 9:45

**Comment by GoogleCodeExporter** [comment]
```
如果安装了Growl，可以在Growl设定介面禁用通知。
参考这一帖：http://tieba.baidu.com/p/2063072980
```

Original comment by `chen....@gmail.com` on 25 Dec 2012 at 10:47
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
谢谢〜

我先去装一个growl好了。mua~
```

Original comment by `5h3l...@gmail.com` on 25 Dec 2012 at 10:51

**Comment by GoogleCodeExporter** [comment]
```
Growl 本身可以设置显示样式、屏蔽消息，只有 Growl 
未运行时必须由输入法控制，所以要用一个三相开关
show_notifications_when: {growl_is_running|always|never}

```

Original comment by `chen....@gmail.com` on 25 Dec 2012 at 11:12

**Comment by GoogleCodeExporter** [comment]
```
fixed in Squirrel 0.9.13
```

Original comment by `chen....@gmail.com` on 27 Dec 2012 at 12:26
- Changed state: **Fixed**
